enterprise/backend/test/metabase_enterprise/serialization/cmd_test.clj

```diff
@@ -77,4 +87,117 @@
                                               (assoc user :password "test-password"))]
           (cmd/load dump-dir "--mode"     :update
                              "--on-error" :abort)
-          (t/is (true? @user-pre-insert-called?)))))))
+          (is (true? @user-pre-insert-called?)))))))
+
+(defn- create! [model & {:as properties}]
+  (db/insert! model (merge (t.test/with-temp-defaults model) properties)))
+
+(defn- do-with-in-memory-h2-db [db-name-prefix f]
+  (let [db-name           (str db-name-prefix (mt/random-name))
+        connection-string (format "jdbc:h2:mem:%s" db-name)
+        data-source       (mdb.data-source/raw-connection-string->DataSource connection-string)]
+    ;; DB should stay open as long as `conn` is held open.
+    (with-open [_conn (.getConnection data-source)]
+      (letfn [(do-with-app-db [thunk]
+                (binding [mdb.conn/*application-db* (mdb.conn/application-db :h2 data-source)]
+                  (println (u/format-color :yellow "USING %s" (pr-str connection-string))) ; NOCOMMIT
```

Comment:
`; NOCOMMIT`. We need the precommit hooks

Reply:
fixed